data/en/isbinary.json

```diff
@@ -17,6 +17,15 @@
 	},
 	"links": [
 
-	]
+	],
 
+	"examples": [
+		{
+			"title":"isBinary example",
```

Comment:
The title is very generic. Could be a bit more descriptive like "Checks if `toBase64()` function returns binary data".